README.md

```diff
@@ -46,7 +46,7 @@ User configuration of the application parameters is done through the ConnectorCo
 
 #### Web Configuration
 
-The ConnectorConfig.json file can be edited through the configuration webpage.  The configuration webpage can be accessed after the [installation process](#Installation) has been completed.  The url for the configuration interface **Your_Flexy's_IP_Address**/usr/config.html
+The ConnectorConfig.json file can be edited through the configuration webpage.  The configuration webpage can be accessed after the [installation process](#Installation) has been completed.  The url for the configuration interface **Your_Flexy's_ip_Address**/usr/config.html
```

Comment:
Why did you lowercase IP? It should be capitalized.

Reply:
This change (if not accidental) does not belong in this commit.

Reply:
change was accidental and is removed